resources/lang/pl_PL/models.yml

```diff
@@ -0,0 +1,57 @@
+generic:
+  createdAt: "utworzono"
+  modifiedAt: "zmodyfikowano"
+  name: "nazwa"
+  email: "adres e-mail"
+  createdBy: "utworzono przez"
+dictionary:
+  _name: "słownik"
+  _names: "słowniki"
+  name: "$t(models.generic.name)"
+  facility: "placówka, właściciel słownika"
+  isFixed: "nieusuwalny / systemowy"
+  positions: "pozycje"
+position:
+  _name: "pozycja słownika"
+  _names: "pozycje słownika"
+  name: "$t(models.generic.name)"
+  dictionary: "słownik"
+  facility: "placówka, właściciel pozycji"
+  isFixed: "nieusuwalna / systemowa"
+  defaultOrder: "domyślna kolejność"
+facility:
+  _name: "placówka"
+  _names: "placówki"
+  name: "$t(models.generic.name)"
+  url: "adres url wewnątrz Memo"
```

Comment:
"adres URL w Memo"? W każdym razie co najmniej "URL".

Ale w ogóle nie jestem pewien czy te tłumaczenia powinny zawierać jakoś dużo objaśnień. Myślę że samo "URL" też by było w porządku, podobnie wyżej te facility.

Reply:
jeżeli to jest to, co użytkownik systemu będzie widział przy polu, to tak, powinien wiedzieć co to jest